.dotnet/src/Custom/Files/FileClient.cs

```diff
@@ -159,10 +123,11 @@ public virtual async Task<ClientResult<OpenAIFileInfoCollection>> GetFileInfoLis
         return ClientResult.FromValue(new OpenAIFileInfoCollection(infoItems), result.GetRawResponse());
     }
 
-    public virtual ClientResult<BinaryData> DownloadFile(string fileId)
+    public virtual ClientResult<Stream> DownloadFile(string fileId)
     {
-        PipelineMessage message = Shim.Pipeline.CreateMessage();
+        using PipelineMessage message = Shim.Pipeline.CreateMessage();
```

Comment:
I noticed that we're not creating message in a `using` block in some places.  We should update this where we're doing it from convenience or protocol methods (but not from request creation helper methods) -- this prevents accidentally not-closing network streams in the case where an exception is thrown before the content stream is buffered.

Reply:
Resolving comment since the experimental "Stream output" approach has been removed from the PR.